radixdlt-core/radixdlt/src/main/java/com/radixdlt/consensus/sync/BFTSync.java

```diff
@@ -249,6 +228,7 @@ public BFTSync(
 						this.vertexStore.highQC().highestCommittedQC(),
 						Optional.of(((FormedTC) viewQuorumReached.votingResult()).getTC()));
 			} else {
+				//TODO: cleanup this mess
 				throw new IllegalArgumentException("Unknown voting result: " + viewQuorumReached.votingResult());
```

Comment:
Should change this to an IllegalStateException for now.

Reply:
done